win_installer.py

```diff
@@ -139,6 +139,29 @@ def about_dict(repo_root):
     return about
 
 
+def pypi_wheel_release(name, version):
+    """
+    Returns a `name==version` string if there's such a wheel at PyPI. When
+    version ends with ".0", a shorter version may be returned, matching what's
+    found at PyPI (name=`PyQtChart` version=`5.12.0` returns `PyQtChart==5.12`,
+    as of this writing). When no releases are found, a ValueError is raised.
+    """
+    package = yarg.get(name)
+    while True:
+        releases = package.release(version)
+        if releases or not version.endswith(".0"):
+            break
+        version = version[:-2]
+
+    if not releases:
+        raise ValueError("No {n!r} package found at PyPI.".format(n=name))
+
+    if any(r.package_type == "wheel" for r in releases):
```

Comment:
I guess the original code already had this issue, but we are only detecting if there any *any* wheels here right?
So if we are building for Windows 32 bits, and only 64 bit wheels are provided (or no windows wheels), we would still think everything is okay.

I guess pynsist will probably check for the right wheels to be available when downloading them, and break at that point, and that to be fair that is fine. If that's the case there is no need to make this script more complicated.

Reply:
Correct. That's due to the way `pynsist` works -- which does feel odd, but it is what it is. We must populate the config file with two sets of dependencies: those available as wheels at PyPI, and the remaining ones.

Reply:
Sounds good.